CONTRIBUTING.md

```diff
@@ -13,15 +13,15 @@ If you are reporting a bug, please include:
 - Any details about your local setup that might be helpful in troubleshooting.
 - Detailed steps to reproduce the bug.
 
+Link to submit <>
+
 ### Fix Bugs
 
-Look through the GitHub issues for bugs. Anything tagged with "bug" and "help
-wanted" is open to whoever wants to implement it.
+Find bugs and fix them!
 
 ### Implement Features
 
-Look through the GitHub issues for features. Anything tagged with "enhancement"
-and "help wanted" is open to whoever wants to implement it.
+Add whatever features you would like to see!
```

Comment:
I think we can keep something closer to the original text, but replacing mentions of "GitHub issues" with our [public-facing Jira board](https://viam.atlassian.net/jira/software/c/projects/RSDK/issues/RSDK-10717?filter=allissues&jql=project%20%3D%20%22RSDK%22%20AND%20component%20%3D%20PythonSDK%20ORDER%20BY%20created%20DESC)